src/gui/windows/workbench.zig

```diff
@@ -47,12 +49,12 @@ var needsUpdate: bool = false;
 fn toggleTool(_: usize) void {
 	currentToolType += 1;
 	currentToolType %= toolTypes.items.len;
-	toolButton.child.label.updateText(toolTypes.items[currentToolType].id);
+	toolButton.child.label.updateText(toolTypes.items[currentToolType].id());
 	needsUpdate = true;
 }
 
 fn openInventory() void {
-	inv = Inventory.init(main.globalAllocator, 26, .{.workbench = toolTypes.items[currentToolType]}, .other);
+	inv = Inventory.init(main.globalAllocator, 26, .{.workbench = .{.index = @intCast(currentToolType)}}, .other);
```

Comment:
No, it must remain an index into the array. Otherwise this will silently break when we implement separate crafting stations.

Also this is relying on an implementation detail of the tool type iterator